media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored the the FLAC format.  flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="http://flac-tools.sourceforge.net/"
+COPYRIGHT="2003-2017 Jake Angerman"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/flac-tools/flac123-$portVersion-release.tar.gz"
+CHECKSUM_SHA256="1976efd54a918eadd3cb10b34c77cee009e21ae56274148afa01edf32654e47d"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```

Comment:
@raefaldhia, this means you just need to:
* remove the line that is defining SECONDARY_ARCHITECTURES, as @korli already wrote.
* remove each occurrence of $secondaryArchSuffix and ${secondaryArchSuffix} because these are unneeded when haikuporter is building for a primary arch. BTW, the recipe works with them because haikuporter simply leaves them undefined, so they count for nothing. But we like to get rid of them when they are not needed.

(Sorry if that was obvious.)

Hmm, I haven't tested the recipe on any arch yet. If you tried on x86 (but not on x86_gcc2) then we first need to make sure it works on x86_gcc2, otherwise you might have to add back the SECONDARY_ARCHITECTURES and the $secondaryArchSuffix and ${secondaryArchSuffix}.